### DISCUSSION TITLE
shellの内部変数

### DISCUSSION THREAD

**Discussion by hayashi-ay** [katataku/minishell#33] [General]
シェルの内部変数をリストアップするスレッド。

**Comment by hayashi-ay** [comment]
主にビルトイン関数で使用することになりそう。

**Comment by hayashi-ay** [comment]
終了ステータス `LAST_EXIT_STATUS`

**Comment by katataku** [comment]
グローバル変数持たせるのはmain.cに統一したいです

**Reply by katataku** [reply]
あーそれか、define.hで宣言だけして、mian関数の冒頭で初期化するか。

**Reply by hayashi-ay** [reply]
それが良いと思います。他のmoduleからも型情報については知ってる必要があるので。
